frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -202,6 +247,10 @@ function ReaderPaging:onSaveSettings()
     self.ui.doc_settings:saveSetting("flipping_scroll_mode", self.flipping_scroll_mode)
 end
 
+function ReaderPaging:onPrevDocument()
+    return self.ui:onOpenLastDoc()
```

Comment:
We did not notice it at the time, but `OpenLastDoc` is not English, it should have been `OpenLastDocument` - but well, not your bad writing.
But `onPrevDocument` is :) Write it `onPreviousDocument` if it is new.